flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/OracleValidator.java

```diff
@@ -65,12 +65,17 @@ public void validate() {
 
     public static Connection openConnection(Properties properties) throws SQLException {
         DriverManager.registerDriver(new oracle.jdbc.OracleDriver());
-        String hostname = properties.getProperty("database.hostname");
-        String port = properties.getProperty("database.port");
-        String dbname = properties.getProperty("database.dbname");
+        String url;
+        if (properties.containsKey("database.url")) {
```

Comment:
we can introduce a jdbc url option in Oracle CDC Connector which will make the table options more precise. Could you also add a test for this PR?

Reply:
Ok, I will supplement the test as soon as possible